src/include/user/abi_dbg.h

```diff
@@ -19,7 +19,7 @@
 #define __USER_ABI_DBG_H__
 
 #define SOF_ABI_DBG_MAJOR 5
-#define SOF_ABI_DBG_MINOR 1
+#define SOF_ABI_DBG_MINOR 2
```

Comment:
@ktrzcinx can you confirm this is a debug ABI update only. If so, we can put in in ABI3.17.

Reply:
This change bumps both ABI numbers